### ISSUE TITLE
Example Notebook Content should be moved to tests

### ISSUE THREAD

**Issue by PipGrylls** [paramm-team/data_processing#24]
Lots of tests are unpopulated and the example notebooks seem to fulfil this need. 

Only consideration is to use cacheing on the imported data object using a fixture OR use a pickle to save the data. Otherwise these tests will all import data and make testing take a long time.

**Comment by PipGrylls** [comment]
#8

**Comment by brosaplanella** [comment]
Agree. We prioritised the examples to demonstrate functionality, and we want to translate them to tests when time allows (probably in the next few weeks). I am not sure how to do the cacheing, could  you write an example of this?

**Comment by PipGrylls** [comment]
Sure thing I’ve not done it before so will be fun to learn On 12 Feb 2024, at 14:31, Ferran Brosa Planella ***@***.***> wrote:﻿
Agree. We prioritised the examples to demonstrate functionality, and we want to translate them to tests when time allows (probably in the next few weeks). I am not sure how to do the cacheing, could  you write an example of this?

**Comment by AlexandruPascu** [comment]
That sounds quite cool, would love to have a chat about it @PipGrylls if you have time. Regarding tests, I agree it'd be nice to populate some more and the notebooks should help as you both said.
